modules/RTC/ScreenObtainer.js

```diff
@@ -94,7 +94,12 @@ const ScreenObtainer = {
      * @param {Object} options - Optional parameters.
      */
     obtainScreenOnElectron(onSuccess, onFailure, options = {}) {
-        if (window.JitsiMeetScreenObtainer && window.JitsiMeetScreenObtainer.openDesktopPicker) {
+        if (typeof window.JitsiMeetScreenObtainer?.openDesktopPicker === 'function') {
+            // Detect if we have the fallback option.
+            if (window.JitsiMeetScreenObtainer?.gDMSupported) {
+                return this.obtainScreenFromGetDisplayMedia(onSuccess, onFailure);
```

Comment:
I just remembered that we apply the min fps constraint on the stream returned by `obtainScreenFromGetDisplayMedia` so that the 0 Hz mode for SS doesn't kick in automatically for browser. Do you think we need to run that on Electron clients as well? I am not sure if that will cause any issues in Electron though.

Reply:
I'd expect not to cause issues since it will be the same code path than for chrome.